lib/chart/line/svg.ex

```diff
@@ -0,0 +1,110 @@
+defmodule Chart.Line.Svg do
+  @moduledoc false
+
+  alias Chart.Line
+  alias Chart.Line.View
+
+  use Phoenix.HTML
+
+  def generate(%Line{} = line) do
+    assigns =
+      line
+      |> Map.from_struct()
+      |> Map.put(:view, View)
+
+    ~E"""
+    <% {width, height} = @settings.figure.viewbox %>
+    <svg version="1.2" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+      viewbox="0 0 <%= width %> <%= height %>" >
+
+      <rect id="figure_bg" width="100%" height="100%" />
+
+      <% {pos_x, pos_y} = @settings.title.position %>
+      <text id="title" x="<%= pos_x %>" y="<%= pos_y %>"
+        alignment-baseline="middle" text-anchor="middle"
+      ><%= @settings.title.text %></text>
+
+      <% {pos_x, pos_y, width, height} = @view.plot_rect_bg(@settings.plot) %>
+      <rect id="plot" x="<%= pos_x %>" y="<%= pos_y %>" width="<%= width %>" height="<%= height %>" />
+
+      <g class="axis">
+        <% x_axis =  @view.get_axis(@settings, {1, 0}) %>
+        <% y_axis =  @view.get_axis(@settings, {0, 1}) %>
```

Comment:
Nemela by axis mit vlastni template a ten sem jen vyspat?

Reply:
Mmm to by mohla :-) to me nenapadlo.